internal/source/prometheus/source.go

```diff
@@ -73,7 +74,14 @@ func (p *Source) consumeAlerts(ctx context.Context, config Config, ch chan<- []b
 			log.Errorf("failed to get alerts. %v", err)
 		}
 		for _, alert := range alerts {
-			msg := fmt.Sprintf("[%s][%s][%s] %s", PluginName, alert.Labels["alertname"], alert.State, alert.Annotations["description"])
+			msg := heredoc.Docf(`
+				Source: %s
+				Alert Name: %s
+				State: %s
+
+				Description:
```

Comment:
What would happen if we used bold for keys? Like:

```suggestion
				*Source:* %s
				*Alert Name:* %s
				*State:* %s

				*Description:*
```

I suspect you already thought of this, and such formatting is not handled correctly in all comm platforms, correct?

Reply:
@pkosiec it becomes bold 
<img width="1197" alt="Screen Shot 2023-02-03 at 10 49 22" src="https://user-images.githubusercontent.com/1237982/216542341-5b697ca1-9ac2-4642-a0d5-148ae3649c0d.png">

Reply:
On Slack and probably Discord + Mattermost yes, but not sure about MS Teams 🤔 I know @mszostok knows a few caveats about MS Teams, that's why I asked 😄 But hopefully we can apply the bold 🤞